scriptmodules/emulators/retroarch.sh

```diff
@@ -110,9 +110,17 @@ function update_core_info_retroarch() {
     local dir="$configdir/all/retroarch/cores"
     # remove if not a git repository and do a fresh checkout
     [[ ! -d "$dir/.git" ]] && rm -fr "$dir"
+    # remove our locally generated `.info` files, just in case upstream adds them
+    local info_file
+    for info_file in "$md_data"/*.info; do
+        rm "$configdir/all/retroarch/cores/$(basename $info_file)" 2>/dev/null
+    done
```

Comment:
What about just doing a `git clean -f -d` ?

Reply:
Yes, that's better. Forgot about `git clean`, I think I used it just a couple of times.
I modified the removal/cleanup for local files to use it.